lib/string_foundation/length.rb

```diff
@@ -0,0 +1,17 @@
+# ==============================================================================
+# LIB - STRING FOUNDATION - LENGTH
+# ==============================================================================
+# frozen_string_literal: true
+class String
+
+  # Compare characters length
+  def length?(length)
+    unless [Integer, Range, Fixnum, Bignum].include?(length.class)
+      raise ArgumentError.new("argument must be Integer (it called Fixnum or Bignum in Ruby 2.3 or lower) or Range")
```

Comment:
How do you feel to use `argument must be Integer (including Fixnum or Bignum) or Range` in this error message?

Reply:
I think it's good.
It's simple and easy to understand.